isis/src/base/objs/CSMCamera/CSMCamera.cpp

```diff
@@ -242,8 +267,35 @@ namespace Isis {
                                     imageMatrix(2,1)};
     return imagePartials;
   }
-}
 
+  // CSMCamera(cube) -> Camera(cube) -> Sensor(cube) -> Spice(cube)
+  // Spice::init() creates a Target in such a way that requires spice data / tables, so 
+  // this works around that.
+  Target *CSMCamera::setTarget(Pvl label) {
```

Comment:
How does this get around the issue of polymorphism in the Spice constructor?

https://stackoverflow.com/questions/1453131/how-can-i-get-polymorphic-behavior-in-a-c-constructor